.prettierignore

```diff
@@ -4,4 +4,7 @@ public/
 .prettierrc.yaml
 pnpm-lock.yaml
 docs/_guides.yaml
+docs/firebase.md
+docs/auth.md
+docs/plugins/firebase.md
```

Comment:
Remove these, you can mark a code block to be ignored from prettier in the markdown file so that you still can get markdown formatting outside of the code block. See other docs for an example of that (search "prettier" in our docs for the proper ignore comment).

Reply:
Thanks for the tip; will do!